src/components/NavBar/Navbar.jsx

```diff
@@ -24,6 +25,11 @@ function Navbar() {
         </button>
         <div className="collapse navbar-collapse" id="navbarNav">
           <ul className="navbar-nav ms-auto">
+          <li className="nav-item">
+              <Link className="nav-link" to="/find-book">
+             <DayNightTheme />
```

Comment:
This is not formatted correctly. Apart from that, clicking a theme toggle should not send me to a different page. If I had written some part of a comment on a book discussion and all that got lost, I would be really upset!